src/Classes/Config.php

```diff
@@ -673,13 +673,24 @@ final class Config
     public static $long_name = 'University Radio York';
     public static $founded = '1967';
     public static $facebook = 'https://www.facebook.com/URY1350';
+
+    /**** SIGNUP EMAILS ****/
+    public static $welcome_email_sender_memberid = null;
     public static $welcome_email = <<<EOT
 
-   This is a welcome email. You can use #NAME to get the user's first name,
-   and include #USER and #PASS to tell them their newly created login details.
+   This is a welcome email. You can use #NAME to get the user's first name.
+
+EOT;
+    public static $account_email = <<<EOT
```

Comment:
this is very generically named, and the documentation below isn't much better. which account? which email? who gives the email to whom?

Reply:
Better? :)